python/ecl/summary/ecl_sum_vector.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, parent, key, report_only = False):
 
         self.__dates = parent.get_dates(report_only)
         self.__days = parent.get_days(report_only)
-        self.__mpl_dates = parent.get_mpl_dates(report_only)
+        self.__mpl_dates = parent.get_numpy_dates(report_only)
```

Comment:
Should not be changed..

Reply:
This I did because I don't want to __init__ __mpl_dates with get_mpl_dates() as this will trigger the Depreciation warning.

Looking at it again I actually suggest to remove self.__mpl_dates completely (as far as I can see it is merely used as a placeholder for the @propery mpl_dates.

So instead I suggest that the @property mlp_dates calls 
`EclSum.get_mpl_dates(report_only)`.

This way the depreciation warning will not be called upon __init__ of the sum vector but only when someone requests the mpl_dates property.

Reply:
I see that EclSumNode also contains the mpl_date (I didn't spot this because I already changed this to numpy_dates). Within the EclSumNode libecl/python/ecl/summary/ecl_sum_node.py the mpl_date doesn't seem to be used.

Any idea if we can simply change this to numpy date as well or would that break other parts?[libecl/python/ecl/summary/ecl_sum_node.py](https://github.com/Statoil/libecl/blob/5a47e7406c944d0667d435e4f489094ba7c15732/python/ecl/summary/ecl_sum_node.py)